SU-SWS/stanford-jumpstart-deployer
# BASIC-1391: Prevent node save when deleting field collections.

What was happening was that anytime a field collection was being deleted from a node it saved the parent node causing all of the node_save hooks to fire. This chain of events lead to pathauto picking up and incorrectly updating or creating a new alias.
- Avoids creating pathauto aliases and other hilarity.
- https://stanfordits.atlassian.net/browse/BASIC-1391
- https://www.drupal.org/node/2385985

To test:
1. Merge
2. Build a dev if https://github.com/SU-SWS/Stanford-Drupal-Profile/pull/58 has been merged or build a production JSA
3. Install JSA and ensure it goes all the way through
4. Create a new node that has a field collection on it
5. Delete that node
6. Check to see that path alias no longer exists.
